src/Lean/Widget/InteractiveCode.lean

```diff
@@ -74,6 +74,8 @@ where
         TaggedText.tag t (go subTt)
 
 def ppExprTagged (e : Expr) (explicit : Bool := false) : MetaM CodeWithInfos := do
+  if pp.raw.get (← getOptions) then
```

Comment:
Does `withOptionAtCurrPos pp.raw true` on the `delab` call not work?

Reply:
`pp.raw` completely bypasses the delaborator.  That's why `pp.raw` is the only that doesn't work.